docs/management/connectors/action-types/servicenow-itom.asciidoc

```diff
@@ -5,7 +5,7 @@
 <titleabbrev>ServiceNow ITOM</titleabbrev>
 ++++
 
-The {sn} ITOM connector uses the https://docs.servicenow.com/bundle/rome-it-operations-management/page/product/event-management/task/send-events-via-web-service.html[Event API] to create {sn} events.
+The {sn} ITOM connector uses the https://docs.servicenow.com/bundle/rome-it-operations-management/page/product/event-management/task/send-events-via-web-service.html[Event API] to create {sn} events. You can use the connector for rule actions.
```

Comment:
Not mandatory, but since you're touching this page anyway it'd be nice to make use of the shared attributes:
```suggestion
The {sn-itom} connector uses the https://docs.servicenow.com/bundle/rome-it-operations-management/page/product/event-management/task/send-events-via-web-service.html[Event API] to create {sn} events. You can use the connector for rule actions.
```